### PR TITLE
Makes Asteroid with Rockets require emagging the console

### PR DESCRIPTION
**Pull request by Ziiro**
:cl: 
add: Asteroid with Engines requires you to emag the coms console before purchase now
/:cl:

also I know it's a config update but for the love of god please also disable imfedupwiththisworld and the luxury shuttle

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Why luxury, it’s ok 

Also
**REEEEEEEEEEEEEEEE**

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -86,10 +86,16 @@
...
 	name = "Asteroid With Engines Strapped To It"
-	description = "A hollowed out asteroid with engines strapped to it. Due to its size and difficulty in steering it, this shuttle may damage the docking area."
+	description = "A hollowed out asteroid with engines strapped to it. Due to its size and poor navigation protocols, this 'shuttle' will damage the docking area."
 	admin_notes = "This shuttle will likely crush escape, killing anyone there."
 	credit_cost = -5000
 
+/datum/map_template/shuttle/emergency/meteor/prerequisites_met(mob/user)
+	if("emagged" in SSshuttle.shuttle_purchase_requirements_met)
+		return TRUE
+	to_chat(user, "<span class='notice'>The console blocks the purchase of that shuttle.</span>")
```
Comment (FlattestGuitar) [inline code/datums/shuttles.dm]: The whole point of the proc existing is to make the messages not vague. It should be some lore-y explanation for the shuttle being present on the list but impossible to buy.

Comment (Ralta) [reply code/datums/shuttles.dm]: Could make it as simple as "Central Command has deemed this shuttle too dangerous for general crew usage." or something similar.

**Comment by Iamgoofball** [comment]
>imfedupwiththisworld 

that shuttle is art, just get with the program

**Comment by Ziiro** [comment]
I mean I get the joke but I just find it to be a really shitty shuttle in so much that it's not really a shuttle at all.

Also have yet to see that shuttle make it to centcom without 90% of people dead or spaced

**Comment by AffectedArc07** [comment]
Ziiro, whats wrong with the luxury shuttle?

**Comment by Ziiro** [comment]
Luxury shuttle fucks a lot of people over as well, but less than Hi Daniel.. The steerage has the same problem though. Grief is rampant.

At least on the normal shuttles you can figure out who's griefing who easier, and people can run to other parts of the shuttle. Such is not the case for those two abominations.

**Comment by AffectedArc07** [comment]
I just played a round with luxury shuttle, I see now.

**Pull request merged**